shell/browser/serial/serial_chooser_controller.h

```diff
@@ -31,7 +32,8 @@ class ElectronSerialDelegate;
 
 // SerialChooserController provides data for the Serial API permission prompt.
 class SerialChooserController final
-    : private SerialChooserContext::PortObserver {
+    : private SerialChooserContext::PortObserver,
+      public device::BluetoothAdapter::Observer {
```

Comment:
Does this need to  be `public`?

Reply:
@ckerr no - I was just matching Chrome here!